test/auth-config.js

```diff
@@ -4,7 +4,7 @@ var Lab = require('lab'),
   test = lab.test,
   expect = require('code').expect;
 
-var auth = require('../lib/auth-config');
+var auth = require('../lib/auth-config')(function() {});
```

Comment:
This is just to ensure the hapi bearer token plugin doesn't throw. These tests only validate the auth config settings, and never calls the validateFunc